beacon-chain/sync/initial-sync/blocks_queue.go

```diff
@@ -0,0 +1,429 @@
+package initialsync
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	eth "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/beacon-chain/blockchain"
+	"github.com/prysmaticlabs/prysm/beacon-chain/p2p"
+	"github.com/prysmaticlabs/prysm/shared/mathutil"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	queueMaxPendingRequests  = 4
+	queueFetchRequestTimeout = 60 * time.Second
+	queueMaxCachedBlocks     = 8 * queueMaxPendingRequests * blockBatchSize
+	queueRefreshRate         = time.Second / 20
+	queueStopCallTimeout     = 1 * time.Second
+)
+
+const (
```

Comment:
Can we add some comments on these enums?

Reply:
Maybe we can define a type for them to use them in func args

Reply:
Yep, definitely, we need type alias here. Will do.